javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -209,7 +209,7 @@ final class ASTParser {
     }
 
     private static class RangedList<T extends Node> {
-        Range range = Range.UNKNOWN;
+        Range range = Range.range(0, 0, 0, 0);
```

Comment:
No wait. We just changed the semantics to unknown = Optional.none(). Can't have two things claiming the same.